README.md

```diff
@@ -9,11 +9,19 @@ getting started for smf - The Fastest RPC In The West
 
 # How-to?
 
-1. ./build.sh
-2. cd build/release
-3. Run the apps
-    *   ./src/demo_server -c 1 & (`run on one core; background`)
-    *   ./src/demo_client -c 1   (`run on one core`)
+Build the demo server and client, which will also build smf and its
+dependencies. **Note that the initial build may take a long time**.
+
+```bash
+cmake .
```

Comment:
we should make it one line ppl can copy paste

`mkdir -p build && cmake .. && make`

Reply:
`make -j$(nproc)